### DISCUSSION TITLE
How do I set up JUnit?

### DISCUSSION THREAD

**Discussion by ghost** [mfussenegger/nvim-jdtls#128] [Q&A]
How do I get rid of the following error? I have junit.jar in my $CLASSPATH but that doesn't seem to do anything. Thanks!
![image](https://user-images.githubusercontent.com/89749779/131264638-1833a7fd-8225-4e32-aa74-98b4f64b6d5e.png)

**Comment by mfussenegger** [comment]
eclipse.jdt.ls requires a maven or gradle project to setup the dependencies. `$CLASSPATH` is afaik not looked at.

You could theoretically also manually create a `.classpath` file under the `-data` workspace folder, but I wouldn't recommend it.
